prow/config.yaml

```diff
@@ -9949,23 +11206,22 @@ periodics:
       secret:
         defaultMode: 256
         secretName: ssh-key-secret
-
-- name: ci-kubernetes-e2e-gke-gpu
-  interval: 2h
+- interval: 6h
```

Comment:
OK.

Might be worth it to have a "prow config diff" tool somewhere, like we have with the old jenkins configs. :-)